Cargo.toml

```diff
@@ -22,7 +22,7 @@ required-features = ["pem"]
 yasna = { version = "0.5", features = ["time", "std"] }
 ring = "0.16"
 pem = { version = "1.0", optional = true }
-time = { version = "0.3", default-features = false }
+time = { version = "~0.3.6", default-features = false }
```

Comment:
The ~ is not really neccessary. In this instance, it is equivalent to writing ^ (or no symbol at all, which is what I suggest).